src/components/lib/cos_defkernel_api.c

```diff
@@ -47,6 +47,8 @@ void
 cos_defcompinfo_init_ext(tcap_t sched_tc, thdcap_t sched_thd, arcvcap_t sched_rcv, pgtblcap_t pgtbl_cap,
                          captblcap_t captbl_cap, compcap_t comp_cap, vaddr_t heap_ptr, capid_t cap_frontier)
 {
+	assert(curr_defci_init_status != UNINITIALIZED);
```

Comment:
No idea where this is defined, but perhaps it should be `DCI_UNITIALIZED` to namespace it.  Don't want a conflict later because it wasn't namespaced.